examples/anonymous-records/anonymous_records.description

```diff
@@ -0,0 +1 @@
+// Record types can be defined in-line as well. These types do not have a type name associated with them.
```

Comment:
Please enhance all descriptions to cover the why aspect and the use case. Apply to all descriptions.

Reply:
Addressed with 8722b1c